### PR TITLE
Adding tests requiring non-empty projectId, clusterId and zoneId.

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableClusterName.java [hunk 1]
```diff
@@ -86,4 +86,22 @@ public void testNoTableName() {
     expectedException.expect(IllegalStateException.class);
     bigtableClusterName.toTableId(clusterName + "/" + BigtableClusterName.TABLE_SEPARATOR);
   }
+
+  @Test
+  public void testNoZoneName() {
+    expectedException.expect(IllegalArgumentException.class);
```
Comment (kevinsi4508) [inline bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableClusterName.java]: expectMessage is better here.

**Comment by kevinsi4508** [comment]
LGTM
